test_data/rules/experiments/nonfunction-type-aliases/rules/avoid_private_typedef_functions.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
```

Comment:
nit: year

Reply:
Since this is a copy of the existing test (w/ one line added for the experiment), I kept the original copyright.  (Once the experiment is enabled by default, this will get folded back into the main test.)

Does that sound right?